### DISCUSSION TITLE
JUnit report and failed screenshots

### DISCUSSION THREAD

**Discussion by CookiesEater** [cypress-io/cypress#16724] [CI setup]
Since Gitlab added attaching screenshots to pipeline (https://docs.gitlab.com/ee/ci/unit_test_reports.html#viewing-junit-screenshots-on-gitlab) I want to use this feature with Cypress in CI.

But for default JUnit report does't contain any information about failed screenshots. Is that possible to add failed screenshots as attachements for mocha-junit-reporter, that used by Cypress to generate JUnit report?

**Comment by Ognengineer** [comment]
bump
